app/controllers/recipes_controller.rb

```diff
@@ -1,7 +1,7 @@
 class RecipesController < ApplicationController
   load_and_authorize_resource
 
-  # before_action :set_recipe, only: %i[show destroy]
+  before_action :set_recipe, only: %i[show destroy]
   def index
     @recipes = Recipe.includes(:user).where(user_id: current_user.id)
```

Comment:
- [ ] Please fix the `index GET` request from the recipes controller because it's throwing this error.

![Screenshot 76](https://user-images.githubusercontent.com/47625407/219825395-286f884f-4b9e-4623-92f4-83e4adae55d0.png)